plugin/octant.d.ts

```diff
@@ -85,6 +87,7 @@ export interface ActionRequest {
 export interface ContentRequest {
   readonly clientID: string;
   readonly contentPath: string;
+  readonly oState: Record<string, any>;
```

Comment:
I know it is a little more effort to keep up-to-date, but I think we know exactly the types that clientState will contain, since we put them on the gRPC wire. Let's capture that in the types for octant.d.ts so authors get type assistance when their editor has it and better enforcement from `tsc`

```
type ClientState struct {
	ClientID    string          `json:"clientID"`
	Filters     []octant.Filter `json:"filters"`
	Namespace   string          `json:"namespace"`
	ContextName string          `json:"contextName"`
}
```